src/app/dashboard-widgets/add-codebase-widget/add-codebase-widget.component.spec.ts

```diff
@@ -89,6 +90,30 @@ describe('AddCodebaseWidgetComponent', () => {
     expect(mockBroadcaster.on).toHaveBeenCalledWith('codebaseDeleted');
   });
 
+  it('should enable buttons if the user owns the space', function(this: TestingContext) {
+    this.testedDirective.userOwnsSpace = true;
```

Comment:
Since this is an `@Input` property, to match the actual scenario as closely as possible they should be injected through the host component template rather than being set this way.